apps/pie-docs/README.md

```diff
@@ -1,25 +1,54 @@
-# pie site
-
-## Helpful Links
-
-## Conventions
-- If you want to create some reusable/shared content, you can add it to a markdown file using the following naming format: `**.content.md`. Eleventy knows to ignore these file types during build, so they will not turn into their own pages. Because we use Nunjucks as our markdown rendering engine, we are able to use includes in our markdown files like so: `{% include 'some/path/to/reusable-stuff.content.md' %}`. This allows us to reuse copy if needed.
+<p align="center">
+  <img align="center" src="src/assets/img/social/landing_home.png" height="200">
+</p>
+
+---
+# PIE Docs
+
+## Getting Started
+- To run the project, start by running `yarn` from the root of the monorepo. Then whilst still at the root run:
+
+  ```
+  yarn dev --filter=pie-docs
+  ```
+
+## Technologies used
+- The website is built with [Eleventy](https://www.11ty.dev/), which is a [Static Site Generator](https://jamstack.org/glossary/ssg/)
+- Content is written in [Markdown](https://www.markdownguide.org/cheat-sheet/) and transformed into `HTML` during build
+- We write templates using the [Nunjucks templating language](https://mozilla.github.io/nunjucks/)
+- For component-like features, we use [shortcodes](https://www.11ty.dev/docs/shortcodes/) written in `Javascript`
+- Styling is written using [Sass](https://sass-lang.com/) using `SCSS` syntax
+- For additional page functionality, we use `Javascript` and load them into the page as [es modules](https://modern-web.dev/guides/going-buildless/es-modules/). There is no `Javascript` build step or bundling (however this could change in the future)
+- Our system tests are written using [Playwright](https://playwright.dev/) and [Percy](https://percy.io/)
+- Our unit tests are written using [Jest](https://jestjs.io/)
+- Our accessibility tests are written using [Axe with Playwright](https://playwright.dev/docs/accessibility-testing)
+
+## Testing
+Our primary concerns are:
+1. Ensuring we never lose any pages
+2. Pages contain correct information and are presented correctly
+3. Prevent easy to catch accessibility violations
+4. Our shortcodes return the expected HTML
+
+### Page testing
+We have route navigation tests that ensure all existing pages can be correctly navigated to. When you add new pages, these tests will fail as there's new unexpected pages (this is by design).
+
+In order to fix the tests, you will need to register the routes to your newly added pages by running: `yarn test:generate-routes`. This will update the json file named `expected-routes.snapshot.json` with the urls to your new page.
```

Comment:
```suggestion
In order to fix the tests, you will need to register the routes to your newly added pages by running: `yarn test:generate-routes`. This will update the JSON file named `expected-routes.snapshot.json` with the URLs to your new page.
```